.mergify.yml

```diff
@@ -169,4 +170,22 @@ pull_request_rules:
       comment:
         message: This PR does not contain release version label after merging.
 
+  # If there is a release version remove 'release version missing' and / or 
+  # release review required
+  - name: release version is a must for merged PRs part 2
+    conditions:
+      - merged
+      - -label=^(release-version)
```

Comment:
this shouldn't be `- label=^(release-version)` - otherwise its negative (not release label and merged) ?

Reply:
Ah yes it should be 
Pr is merged and a label beginning with 'release-version' is present. I missed the extra '-'

Reply:
Should now be fixed